app/graphql/mutations/profile/edit.rb

```diff
@@ -17,7 +17,10 @@ def resolve(args = {})
           ::Profile.find(args[:id]),
           :edit?
         )
-        profile.update(args.to_h)
+        profile.update(
+          compliance_threshold: args[:compliance_threshold],
```

Comment:
We could also use named args in this resolve method to clean up the code a bit. (optional)

Reply:
I realized we cannot use named args here, because they force the arguments to exist. Therefore, a mutation that contains "ID" and "threshold" but not "businesss objective ID" does not work. All three should exist.